Example/Tests/Specs/Extensions/Array+RemoveSpec.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Array+RemoveSpec.swift
+//  Tests
+//
+//  Created by Jelle Vandebeeck on 28/11/2018.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Quick
+import Nimble
+
+@testable import Stella
+
+class ArrayRemoveSpec: QuickSpec {
+    override func spec() {
+        describe("Array+Remove") {
+            it("should return the index for the given element") {
```

Comment:
Misschien een domme vraag, maar wat als er 2x de waarde inzit? Gaat die enkel de laatste index teruggeven?

Reply:
Klopt, maar dat is eigenlijk data dat fout zit.

Reply:
Kunnen we later nog iets voor voorzien indien dit een issue blijkt te zijn.

Reply:
Ok!